pkg/commands/flagsets.go

```diff
@@ -47,9 +48,13 @@ func setupRunFlagSet(v *viper.Viper, fs *pflag.FlagSet) {
 	internal.AddFlagAndBind(v, fs, fs.Duration, "timeout", "run.timeout", defaultTimeout, color.GreenString("Timeout for total work"))
 
 	internal.AddFlagAndBind(v, fs, fs.Bool, "tests", "run.tests", true, color.GreenString("Analyze tests (*_test.go)"))
+
+	fs.StringSlice("skip-files", nil, color.GreenString("Regexps of files to skip")) // Hack see Loader.applyStringSliceHack
+	deprecateFlag(fs, "skip-files")
 	fs.StringSlice("skip-dirs", nil, color.GreenString("Regexps of directories to skip")) // Hack see Loader.applyStringSliceHack
+	deprecateFlag(fs, "skip-dirs")
 	internal.AddFlagAndBind(v, fs, fs.Bool, "skip-dirs-use-default", "run.skip-dirs-use-default", true, getDefaultDirectoryExcludeHelp())
-	fs.StringSlice("skip-files", nil, color.GreenString("Regexps of files to skip")) // Hack see Loader.applyStringSliceHack
+	deprecateFlag(fs, "skip-dirs-use-default")
```

Comment:
Perhaps the following is better because it reduces duplication of name:
```go
	internal.AddDeprecatedFlag(fs, fs.StringSlice, "skip-files", nil, color.GreenString("Regexps of files to skip"))
	internal.AddDeprecatedFlag(fs, fs.StringSlice, "skip-dirs", nil, color.GreenString("Regexps of directories to skip"))
	internal.AddDeprecatedFlagAndBind(v, fs, fs.Bool, "skip-dirs-use-default", "run.skip-dirs-use-default", true,
		getDefaultDirectoryExcludeHelp())
```

But we need to extend `pkg/commands/internal/vibra.go` with the following functions:
```
func AddDeprecatedFlag[T any](fs *pflag.FlagSet, pfn FlagFunc[T], name string, value T, usage string) {
	pfn(name, value, usage)
	deprecateFlag(fs, name)
}

func AddDeprecatedFlagAndBind[T any](v *viper.Viper, fs *pflag.FlagSet, pfn FlagFunc[T], name, bind string, value T, usage string) {
	AddFlagAndBind(v, fs, pfn, name, bind, value, usage)
	deprecateFlag(fs, name)
}

func deprecateFlag(fs *pflag.FlagSet, name string) {
	_ = fs.MarkHidden(name)
	_ = fs.MarkDeprecated(name, "check the documentation for more information.")
}
```

Reply:
ok for `AddDeprecatedFlagAndBind` but I don't want to factorize around "hacked" flags.

Reply:
I found a difference approach that allows both: visible Hack and factorization.